mpt/impl/imap-mailbox/core/src/main/resources/org/apache/james/imap/scripts/UidSearchCombinations.test

```diff
@@ -632,7 +632,7 @@ C: Rhubarb!
 S: \* 9 EXISTS
 S: \* 9 RECENT
 S: A72 OK (\[.+\] )?APPEND completed\.
-C: A73 APPEND testmailbox {11976+}
+C: A73 APPEND testmailbox {11975+}
```

Comment:
Well, you are changing a test suite for conveniance.

However, how do you know a MUA (for example Thunderbird) do not use 11976 instead of your value 11975 ? How can you be sure there is no regression in the quality of the test suite ?

Any way, I believe using a locally wired MPT will solve much of your troubles.